src/operationsClient.ts

```diff
@@ -162,7 +162,19 @@ export class OperationsClient {
   }
 
   // Service calls
-
+  getOperationInternal(
+    request: protos.google.longrunning.GetOperationRequest,
+    options?: gax.CallOptions,
+    callback?: Callback<
+      protos.google.longrunning.Operation,
+      protos.google.longrunning.GetOperationRequest,
+      {} | null | undefined
+    >
+  ): CancellablePromise<ResultTuple> {
+    request = request || {};
```

Comment:
As marked, `request` is required so theoretically it shouldn't be null....   but maybe we should just flag it as optional :)  

Aside - I find it cleaner to do the `|| {}` in the function parameters like this:


```js
getOperationInternal(
  request: protos.google.longrunning.GetOperationRequest = {},
  options?: gax.CallOptions = {},
  callback?: ....
);
```

It treats the argument as optional, but with a default value if none is passed.

Reply:
Thanks for the suggestion! `request` is required here, but just in case it's null, so we do `request = request || {}`, but `request: protos.google.longrunning.GetOperationRequest = {}` makes sense to me totally. Refactoring now.